src/OpenDebugAD7/OpenDebugAD7.csproj

```diff
@@ -82,6 +82,10 @@
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
+	<Reference Include="Microsoft.VisualStudio.OLE.Interop">
+		<HintPath>$(NuGetPackagesDirectory)/Microsoft.VisualStudio.OLE.Interop.16.7.30328.74/lib/net45/Microsoft.VisualStudio.OLE.Interop.dll</HintPath>
```

Comment:
You are missing the package reference for us to pull this in.
```
<package id="Microsoft.VisualStudio.OLE.Interop" version="16.7.30328.74" />
```
in 
https://github.com/microsoft/MIEngine/blob/1ccfd123289100f2415450d68b6011766941cc3d/src/OpenDebugAD7/packages.config#L9

The build machine is using the one from MSBuild rather than a specific version:
```
Copying file from "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\PublicAssemblies\Microsoft.VisualStudio.OLE.Interop.dll" to "D:\a\1\s\bin\Desktop.Debug\Microsoft.VisualStudio.OLE.Interop.dll".
```